repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -412,6 +417,12 @@ export interface FullTimeRestrictionAttribute {
     id?: number;
 }
 
+export interface FullCapAttribute {
+    fullCaps: CapInfo[];
+    errors: ErrorInfo[];
+    id?: number;
+}
+
```

Comment:
Im trying to understand what all the new types are for.

I thought that all we would need to store in the session would be the cap they chose (e.g. { id: 2 } ), and that session attribute could also be used to store errors if needed.